rate/rate.py

```diff
@@ -0,0 +1,228 @@
+# coding: utf-8
+"""
+This is an XBlock designed to allow people to provide feedback on our
+course resources.
+"""
+
+import random
+
+import pkg_resources
+
+from xblock.core import XBlock
+from xblock.fields import Scope, Integer, String, List, Float
+from xblock.fragment import Fragment
+
+try:
+    from eventtracking import tracker
+except ImportError:
+    class tracker(object):  # pylint: disable=invalid-name
+        """
+        Define tracker if eventtracking cannot be imported. This is a workaround
```

Comment:
For a while, I maintained a repo which contained common hack-arounds, but we've gradually pushed enough fixes that it stopped being worthwhile. 

In this case, the proper fix would be in edx-platform. There isn't much that could be done in the xblock repo.